### ISSUE TITLE
PRIDE Ontology terms not included into PSI-MS

### ISSUE THREAD

**Issue by ypriverol** [HUPO-PSI/psi-ms-CV#10]
@germa @edeutsch : 

Can we make sure the following files are included in the PSI-MS ontology, for the file types: 

- [ ] Quantification file URI 
- [ ] FASTA file URI 
- [ ] Spectrum Library file URI 
- [ ] MS Image file URI 

These terms are used in PRIDE submission pipeline, would be great to have them.

**Comment by edeutsch** [comment]
Are these going next to existing similar terms?

One concern I have is that we are multiplying two concepts here. One concept is a file type and the other concept is a file URI. Do we need a separate URI term for each file type? What about an mzML file URI? A X!Tandem parameter file URI? The list could go on.. Seems like this would benefit from two separate CV terms. One CV param is the file type (of which we already have many, right?) and the second CV Param is a file URI. I'm not sure what the use case is here, but this seems a little more sensible than combinatorially multiplying them?

**Comment by ypriverol** [comment]
I agree with you. The current URI looks like a classification of different types of data submitted to PX. Gel, result, search, etc. Since long time ago pride is also handling other three categories that are Fasta databases, spectrum library and quant results. All these files are now classify as additional files URI in ProteomeXchange.. but they shouldn’t. That is the main reason why I’m asking for them in PSI-MS.
